app/src/main/res/values-ar/strings.xml

```diff
@@ -202,6 +202,7 @@
     <string name="highlight_weekends">ظلل نهايات الأسبوع في بعض العروض</string>
     <string name="allow_changing_time_zones">Allow changing event time zones</string>
     <string name="manage_quick_filter_event_types">Manage quick filter event types</string>
+    <string name="add_bdays_anniv_automatically">Add birthdays and anniversaries automatically after launching the app</string>
```

Comment:
let's change this to "Add new birthdays automatically" and "Add new anniversaries automatically"